android-components/components/feature/session/src/main/java/mozilla/components/feature/session/behavior/EngineViewBrowserToolbarBehavior.kt

```diff
@@ -41,7 +41,9 @@ class EngineViewBrowserToolbarBehavior(
     @VisibleForTesting
     internal var toolbarChangedAction: (Float) -> Unit?
     private val bottomToolbarChangedAction = { newToolbarTranslationY: Float ->
-        engineView?.setVerticalClipping(-newToolbarTranslationY.roundToInt())
+        if (!newToolbarTranslationY.isNaN()) {
```

Comment:
is it possible to also add a test for this condition?

Reply:
I'll check into it